src/shared/prerendered-app/Intro/style.css

```
@@ -118,7 +118,115 @@
   fill: var(--light-blue);
 }
 
+.bottom-wave {
+  position: relative;
+}
+
+.info {
+  background: var(--white);
+  position: relative;
+  padding: 5em 2em;
+  &:nth-of-type(even) {
+    & .info-img-wrapper {
+      order: 1;
+      @media (min-width: 768px) {
+        justify-self: start;
+      }
+    }
+
+    & .info-text-wrapper {
+      order: 2;
+      text-align: right;
+      @media (min-width: 768px) {
+        justify-self: end;
+      }
+    }
+  }
+
+  @media (min-width: 1200px) {
+    padding: 5em 0;
+  }
+}
+
+.info-container {
+  max-width: 1000px;
+  margin: 0 auto;
+  opacity: 0;
+  @media screen and (prefers-reduced-motion: no-preference) {
+    transform: translate(0px, 40px);
+    transition: opacity 0.3s ease, transform 0.3s ease;
+  }
+}
+
+.info-container.appear {
+  opacity: 1;
+  @media screen and (prefers-reduced-motion: no-preference) {
+    transform: translate(0px, 0px);
+  }
+}
+
+.info-content {
+  display: grid;
+  align-items: center;
+  grid-template-columns: repeat(auto-fit, minmax(min(100%, 320px), 1fr));
+  gap: 1em;
+}
+
+.info-title {
+  color: var(--pink);
+  font-size: 3em;
+  margin: 0;
+}
+
+.info-caption {
+  font-size: 1.5em;
+  line-height: 1.75;
+  margin: 1em 0 0.5em 0;
+}
+
+.info-link {
+  font-size: 1.25em;
+  color: var(--off-black);
+  transition: color 400ms ease-in-out;
```

```suggestion
  transition: color 400ms ease-in-out;
  margin-top: 1em;
```

adds some space above the learn more link